lib/events.js

```diff
@@ -160,7 +160,7 @@ EventEmitter.prototype.emit = function emit(type) {
       throw er; // Unhandled 'error' event
     } else {
       // At least give some kind of context to the user
-      var err = new Error('Uncaught, unspecified "error" event. (' + er + ')');
```

Comment:
I'm not sure it's a good idea to remove the last part. I understand that if you emit a plain object, it will print `[object Object]` but it seems logical for the "error" event to emit an `Error` and in this case it would print the error message correctly.